db/migrate/20210407133724_rename_city_to_train_station_in_users.rb

```diff
@@ -0,0 +1,5 @@
+class RenameCityToTrainStationInUsers < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :users, :city, :train_station
```

Comment:
I might have misunderstood, but from 2 meetings before, I thought it as decided that `city` would be rename to `train_station` for both the user and the group?

Reply:
yes, that is correct, we do want that
but if you go to the controller which params are supported?
if you go to the model which validations are setup?
that's what I'm saying - you are just changing the column name without realising the impact it will have in other places
so run `rspec` and fix all the places that need fixing